docs/guide/snaps.md

```diff
@@ -114,11 +114,11 @@ Make sure to install Flask in a new browser profile or disable any existing inst
 
 Once you have [MetaMask Flask](https://metamask.io/flask/) installed (in a new browser profile _without MetaMask stable_) and the [development environment](./snaps.html#quick-start-using-our-template) running, you should be able to **connect** and **install** the snap.
 
-1. Click the **Connect-button** and the MetaMask Flask extension should pop up and require you to approve the template snap's permissions.
+1. Click the **Connect** button and the MetaMask Flask extension should pop up and require you to approve the template snap's permissions.
 
-2. Once connected, try out the **Send message-button** to display a custom message within a confirmation screen in MetaMask.
+2. Once connected, try out the **Send message** button to display a custom message within a confirmation screen in MetaMask.
 
-And voilà, you've successfully connected, installed, and interacted with your snap.
```

Comment:
For the sake of the various levels of English speakers who may read these docs, we should not use non-English idioms in our documentaiton. 🎻